shuup/admin/static_src/base/js/select.js

```diff
@@ -20,7 +20,7 @@ function activateSelect($select, model, attrs={}) {
             url: window.ShuupAdminConfig.browserUrls.select,
             dataType: "json",
             data: function(params) {
-                return {model: model, search: params.term};
+                return {model: model, searchMode: searchMode, search: params.term};
```

Comment:
can this be `{model: model, searchMode, search: params.term}`

Reply:
I think not. At least the eslint complains about it.